### DISCUSSION TITLE
Minimal example for LiteLLM with using GPT-4[T] vs. a model that doesn't support functions calling and needs prompt wrapping?

### DISCUSSION THREAD

**Discussion by twardoch** [567-labs/instructor#475] [General]
Would anyone care to contribute a minimal Python example for LiteLLM with using GPT-4 or GPT-4Turbo (that supports function calling)  vs. a model that doesn't support functions calling and needs prompt wrapping?

Ideally starting from something like a pydantic model.

**Comment by jxnl** [comment]
https://jxnl.github.io/instructor/concepts/patching/
